emop-maven-plugin/src/main/java/edu/cornell/ImpactedHybridMojo.java

```diff
@@ -30,9 +30,12 @@ public class ImpactedHybridMojo extends HybridMojo {
      */
     protected boolean debug;
 
+    protected boolean computeAffectedTests;
```

Comment:
@pengyuejiang and I thought that this flag is for deciding whether to additionally compute affected classes. But the name suggests that you are computing affected tests. Can you please clarify? Also, can you please point us to a usage of this field?